MaxText/configs/base.yml

```diff
@@ -42,6 +42,9 @@ metrics_file: "" # for testing, local file that stores scalar metrics. If empty,
 # If true save metrics such as loss and TFLOPS to GCS in {base_output_directory}/{run_name}/metrics/
 gcs_metrics: False
 
+# If true save train config to {base_output_directory}/{run_name}/
+gcs_train_config: False
```

Comment:
This change is very reasonable but should we just print to stdout (as well or maybe instead?) Like for 90% of use cases that is what the people need.

Reply:
actually I'll take care of this, NVM